### DISCUSSION TITLE
Crash on opening

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21182] [Q&A]
### Type

Crash to desktop

### Bug description

It crashes upon opening (you see "welcome to lazer" or whatnot) and sometimes if you get past that it crashes upon logging in.

disclaimer: i am reporting this for someone else

### Screenshots or videos

N/A

### Version

2022.1101.0

### Logs

[logs.zip](https://github.com/ppy/osu/files/9966428/logs.zip)

**Comment by Axilotl17** [comment]
is working now (we changed nothing)

**Comment by Axilotl17** [comment]
I take it back
now its freezing
we tried to delete all osu files, nothing worked

**Comment by peppy** [comment]
It's going to be a GPU or driver issue. Please update drivers.
